upgrading/automated_upgrades.adoc

```diff
@@ -732,6 +732,23 @@ other nodes in their class (`app` versus `infra`), one at a time.
 
 . Upgrade the {product-title} nodes running GlusterFS one at a time.
 
+.. Add the following parameters in the inventory file at *_/etc/ansible/hosts_*:
++
+----
+openshift_storage_glusterfs_version=v<your_cns_version> <1>
+openshift_storage_glusterfs_block_version=v<your_cns_version>
+openshift_storage_glusterfs_heketi_version=v<your_cns_version>
+----
```

Comment:
@lpettyjo From QE side, we didn't test the above config in the inventory file. We usually set the glusterfs images and version with the following in the inventory file:
```
  openshift_storage_glusterfs_heketi_image: registry.access.redhat.com/rhgs3/rhgs-volmanager-rhel7:version
  openshift_storage_glusterfs_image: registry.access.redhat.com/rhgs3/rhgs-server-rhel7:version
  openshift_storage_glusterfs_block_image: registry.access.redhat.com/rhgs3/rhgs-gluster-block-prov-rhel7:version
  openshift_storage_glusterfs_s3_image: registry.access.redhat.com/rhgs3/rhgs-s3-server-rhel7:version
```

Reply:
@qinpingli are you suggesting that instead of adding the parameters that I'm proposing (lines 738-740) to the inventory file that instead we instruct users to add the four lines you list above? If so, do the 4 lines you list accomplish the same thing? Sorry for the late reply!

Reply:
@lpettyjo Yes, this is the inventory file I used to define the image version:
```
openshift_hosted_registry_storage_kind=glusterfs
openshift_storage_glusterfs_heketi_image=registry.access.redhat.com/rhgs3/rhgs-volmanager-rhel7:v3.11.2
openshift_storage_glusterfs_image=registry.access.redhat.com/rhgs3/rhgs-server-rhel7:v3.11.2
openshift_storage_glusterfs_block_image=registry.access.redhat.com/rhgs3/rhgs-gluster-block-prov-rhel7:v3.11.2
openshift_storage_glusterfs_s3_image=registry.access.redhat.com/rhgs3/rhgs-s3-server-rhel7:v3.11.2
```

These are the images got from the cluster:
```
# oc get pod -n glusterfs -oyaml|grep "image:"
      image: registry.access.redhat.com/rhgs3/rhgs-gluster-block-prov-rhel7:v3.11.2
      image: registry.access.redhat.com/rhgs3/rhgs-gluster-block-prov-rhel7:v3.11.2
      image: registry.access.redhat.com/rhgs3/rhgs-server-rhel7:v3.11.2
      image: registry.access.redhat.com/rhgs3/rhgs-server-rhel7:v3.11.2
      image: registry.access.redhat.com/rhgs3/rhgs-server-rhel7:v3.11.2
      image: registry.access.redhat.com/rhgs3/rhgs-server-rhel7:v3.11.2
      image: registry.access.redhat.com/rhgs3/rhgs-server-rhel7:v3.11.2
      image: registry.access.redhat.com/rhgs3/rhgs-server-rhel7:v3.11.2
      image: registry.access.redhat.com/rhgs3/rhgs-volmanager-rhel7:v3.11.2
      image: registry.access.redhat.com/rhgs3/rhgs-volmanager-rhel7:v3.11.2
```